hack/jenkins/test-image/run.sh

```diff
@@ -42,6 +38,10 @@ export KUBE_COVERPROCS=8
 export KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4
 export LOG_LEVEL=4
 
+./hack/build-go.sh
+godep go install ./...
+./hack/install-etcd.sh
```

Comment:
another option (maybe this is getting into too much indirection?) is that this file calls hack/jenkins/gotest.sh (or equivalent).

the advantage there is that the list of tests run follows the branch, rather than using the same list across all builds. using the same run.sh for all branches means that we'll have to have special cases.